[2024-06-11T06:28:00.566+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-11T06:28:00.585+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_word.read_files_from_path manual__2024-06-11T06:27:59.556582+00:00 [queued]>
[2024-06-11T06:28:00.592+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_word.read_files_from_path manual__2024-06-11T06:27:59.556582+00:00 [queued]>
[2024-06-11T06:28:00.593+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-11T06:28:00.605+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): read_files_from_path> on 2024-06-11 06:27:59.556582+00:00
[2024-06-11T06:28:00.611+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2634) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-11T06:28:00.613+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hello_word', 'read_files_from_path', 'manual__2024-06-11T06:27:59.556582+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/simple_hello_world.py', '--cfg-path', '/tmp/tmpjthn513b']
[2024-06-11T06:28:00.621+0000] {standard_task_runner.py:63} INFO - Started process 2635 to run task
[2024-06-11T06:28:00.627+0000] {standard_task_runner.py:91} INFO - Job 115: Subtask read_files_from_path
[2024-06-11T06:28:00.679+0000] {task_command.py:426} INFO - Running <TaskInstance: hello_word.read_files_from_path manual__2024-06-11T06:27:59.556582+00:00 [running]> on host 678b3b027d57
[2024-06-11T06:28:00.758+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Anudeep' AIRFLOW_CTX_DAG_ID='hello_word' AIRFLOW_CTX_TASK_ID='read_files_from_path' AIRFLOW_CTX_EXECUTION_DATE='2024-06-11T06:27:59.556582+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-11T06:27:59.556582+00:00'
[2024-06-11T06:28:00.759+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-11T06:28:00.777+0000] {base.py:84} INFO - Using connection ID 'invoices' for task execution.
[2024-06-11T06:28:00.781+0000] {python.py:237} INFO - Done. Returned value was: ['/opt/***/dags/invoices/sample1.json', '/opt/***/dags/invoices/sample2.json']
[2024-06-11T06:28:00.782+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-11T06:28:00.805+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hello_word, task_id=read_files_from_path, run_id=manual__2024-06-11T06:27:59.556582+00:00, execution_date=20240611T062759, start_date=20240611T062800, end_date=20240611T062800
[2024-06-11T06:28:00.842+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-11T06:28:00.891+0000] {taskinstance.py:3503} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-06-11T06:28:00.897+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
